src/app/components/change-email/change-email.component.html

```diff
@@ -0,0 +1,61 @@
+<mat-drawer-container class="example-container">
+  <mat-drawer mode="side" opened>
+    <mat-nav-list>
+        <a mat-list-item routerLink="/settings"> Overview </a>
+        <a mat-list-item> Change Email </a>
```

Comment:
This has no routerLink

Reply:
It shouldn't have one there since that's it's current tab.